quiz-1 solution-3.ipynb

```diff
@@ -0,0 +1,114 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "eefcb042-8730-4ed6-8358-eef9f18415b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename=\"c://sqlite3//BCA.txt\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "786714fd-98e2-4764-a7d4-fddb7550948a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "One line added successful\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open(filename,\"w\",newline=\"\")as write_file:\n",
+    "    heading=['\\t\\tDetils About BCA Course']\n",
+    "    write_file.writelines(heading)\n",
+    "    print('One line added successful')\n",
+    "    content=['BCA is computer science Course.\\n',\n",
+    "             'It is stand for Bachelor of computer application\\n',\n",
+    "             'In this course,we learn about programming languages and related to computer knoweldge\\n',\n",
+    "             'BCA is 3 years of course,we can also clear BCA honur with 4 year.',\n",
+    "             'After BCA Many we can work in many companies work as a software developer,application developer,web designer and lots of jobs.'\n",
+    "            ]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "1014c94d-b663-45a1-be60-2fb12372a3e6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\t\tDetils About BCA Course\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open(filename,\"r\")as read_file:\n",
+    "    data=read_file.read()\n",
+    "    print(data)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "b411b812-ef31-41b6-943c-53ed5e8b89bd",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total  in text file is: 1\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open(filename,\"r\")as read_file:\n",
+    "    data=read_file.read()\n",
+    "    count=0\n",
+    "    for i in range(len(data)):\n",
+    "        if data[i][0]==str('B'):\n",
```

Comment:
Please read the program definition again it say that write lines which starts with 'a' , 'b' ,'c' to another file and all write total lines in the another file